src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+export const addbook = (book) => ({
+  type: ADD_BOOK,
+  payload: book,
+});
+
+export const removeBook = (book) => ({
+  type: REMOVE_BOOK,
+  payload: book,
+});
+
+const bookReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.payload.id);
+    default:
+      return state;
+  }
```

Comment:
- [ OPTIONAL ] Great job so far @brainskev. Your app works great, I will suggest that you make use of `createSlice` from `@reduxjs/toolkit` as it provides several advantages over the `traditional switch statement syntax `used to define a reducer function. :+1: 
- Here are some reasons why using `createSlice` is a better approach:

1. Less Boilerplate: `createSlice` reduces the amount of boilerplate code needed to create a reducer function by handling the creation of the action creators and reducer logic in one step. This makes it quicker and easier to write and maintain code. :100: 

2. Immutable Updates: `createSlice` uses `Immer` internally to handle immutable updates to the state. This means that you can write code that looks like you're directly modifying the state, but it creates a new immutable copy of the state. This helps avoid accidental mutations of state and makes it easier to reason about the application's state. This is something you will need in future projects to ensure that the state of your app is immutable :+1: 

Overall, using `createSlice` from `@reduxjs/toolkit `can lead to more concise, maintainable, and performant code. :100: You can learn more about it [here](https://redux-toolkit.js.org/api/createSlice) :wink: